tests/debugger/test_debugger_probe_snapshot.py

```diff
@@ -133,3 +148,33 @@ def test_code_origin_entry_present(self):
                 code_origins_entry_found = code_origin_type == "entry"
 
         assert code_origins_entry_found
+
+    def setup_log_line_probe_snaphots_budgets(self):
+        self._setup(
+            "probe_snapshot_log_line_budgets",
+            "/debugger/budgets/150",
+            lines=self.method_and_language_to_line_number("Budgets", self.get_tracer()["language"]),
+        )
+
+    @features.debugger_probe_budgets
```

Comment:
Do we need to declare 'budget' as a feature? I think it is part of the dynamic instrumentation product and not a feature on its own. If yes, I believe we should add RFC and doc links at the feature parity dashboard.

Reply:
I think so, it's fairly complicated with dynamic instrumentation budgets, live debugging budgets, and the idea around max thresholds